docs/upstream-differences.md

```diff
@@ -88,3 +88,70 @@ Once installer fork is ready:
 1. After this point, you should be able to create a dev cluster using the RP and it should use the new release.
 1. `make discoverycache`.
     * This command requires a running cluster with the new version.
+
+
+# Namespaces that ARO-RP uses mainly
+
+The namespaces are put in alphabetical order below: 
+```
+openshift
```

Comment:
I would remvoe the list of namespaces the the docs: we don't want to update two places (code and docs) manually.

But we should add a step under "Update ARO-RP" saying that the list in `pkg/util/namespace/namespace.go` needs to be updated as part of introduction of a new minor OCP version as suggedted by Ben [here](https://github.com/Azure/ARO-RP/pull/1994#pullrequestreview-900381460).